assets/_scss/elements/_typography.scss

```diff
@@ -17,12 +17,22 @@ p {
 }
 
 a {
-  color: $color-base-link;
-  &:hover, &:active, &:visited {
-    color: $color-base-link;
+  border-bottom: 1px dotted;
+  &:hover {
+    border-bottom: 1px solid;
   }
 }
 
+.usa-link {
+  border-bottom: none;
+  font-family: $font-sans; 
```

Comment:
the normal `<a>` tag has has a border-bottom and serif style, which needs to get removed for the alternate sans-serif link.